### DISCUSSION TITLE
[@types/forge-apis] Missing important functions for model derivatives

### DISCUSSION THREAD

**Discussion by GoGoPen** [DefinitelyTyped/DefinitelyTyped#58271] [Issues with a @types package] [answered]
I am trying to convert the Forge Viewer with model derivatives from Express.js to Nest.js.

However, I found that JobSvfOutputPayload.js, JobSvfOutputPayloadAdvanced.js, JobSvf2OutputPayload.js, JobSvf2OutputPayloadAdvanced.js are missing from the @types/forge-apis. 
I think that this is an important function to convert the various bim model into svf/svf2 format such that we could view the bim model in forge viewer. 

Any clue how to add them to the @types/forge-apis ? Thanks.

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
PR is ready #58282

**Reply by GoGoPen** [reply]
Thanks!
